core/modules/util/threadSafe.h

```diff
@@ -73,21 +73,23 @@ class Flag {
 
     /** Sets flag value to 'val' and returns the old value of flag.
      */
-    virtual T set(T val) {
+    virtual T exchange(T val) {
```

Comment:
For the record - I do not like this class :smile_cat: It should be removed, together with Sequential.